src/app/contexts/UserContext/cookies/index.js

```diff
@@ -1,17 +1,27 @@
 import Cookie from 'js-cookie';
+import { getVariantCookieName } from '#lib/utilities/variantHandler';
 
 export const getCookiePolicy = () => {
   const POLICY_COOKIE = 'ckns_policy';
 
   return Cookie.get(POLICY_COOKIE) || '000';
 };
 
+export const personalisationEnabled = cookiePolicy =>
+  cookiePolicy && cookiePolicy.length === 3 && cookiePolicy[2] === '1';
+
 export const getPreferredVariant = service => {
   if (!service) return null;
-  const VARIANT_COOKIE = `ckps_${service}`;
+  const VARIANT_COOKIE = `ckps_${getVariantCookieName(service)}`;
```

Comment:
Thanks for moving it to a utilities file. That's much better than the services config. 
I think it'd be better for the variant cookie to be in the shared utility file instead of interpolated like this.

E.g.
```
const variantCookie = {
  serbian: 'ckps_serbian', 
  ukchina: 'ckps_chinese', 
  zhongwen: 'ckps_chinese', 
} ;
```
With this exported from the utility file and used below as `variantCookie(service)`.
Happy for this refactor to be done later though in the interest of getting this implementation on the Test environment ASAP to get testing the cookie behaviour on `www.test.bbc.com`.